pkg/artifact/kit-file.go

```diff
@@ -79,11 +80,23 @@ type (
 )
 
 func (kf *KitFile) LoadModel(filePath string) error {
-	modelfile, err := os.Open(filePath)
-	if err != nil {
-		return err
+	var modelfile io.ReadCloser
+	if filePath == "-" {
+		stat, _ := os.Stdin.Stat()
+		if (stat.Mode() & os.ModeCharDevice) == 0 {
+			modelfile = os.Stdin
+		} else {
+			output.Errorln("No input file specified and no data piped")
+		}
```

Comment:
Should this just return an error? In addition, I don't think library-side functions should be doing output (e.g. if someone later wants to import and use LoadModel in another package, using the output package will probably break something). 

Since we don't return here, the later call to `io.ReadAll` could still hang indefinitely.

Reply:
yes makes sense. I moved this part of the implementation looks like I forgot this here. Should be fixed now